cfgov/v1/jinja2/v1/story_page.html

```diff
@@ -17,6 +17,7 @@
         {% endif %}
     {% endfor %}
 {% endblock %}
+{% block pre_content -%}{%- endblock %}
```

Comment:
Adding `pre_content` removes the breadcrumbs? Is this right? Manually following the chain I see `pre_content` in `content-base.html` and it's a block for the breadcrumbs.

Reply:
yep, the only purpose of the `pre_content` block in `content_base` is to add the breadcrumbs. Overriding that block in the child template removes the breadcrumbs.

Reply:
Oh oh right. I was thinking it was making it available here, but right, this clears it. You could consider having `story_page.html` extend directly from `base.html` also. It may result in some minor duplication, but that seems preferable to the long chain through story_page > layout-full > content-base. Without the breadcrumbs, those intermediary templates provide very little.

Reply:
Yes, without understanding what else those intermediary templates are providing, it feels a little magic to hide the breadcrumbs by adding an empty block. Someone coming to this code in future likely wouldn't understand that's what this is for. If we go this approach, could you consider adding a `{# comment #}` indicating the purpose for the empty block? Ideally it'd be nice to have a unit test that verified that StoryPages don't show breadcrumbs, too, as extra confirmation in case this template logic ever gets modified.

Reply:
Comment added!

Reply:
Test added as well~!

Reply:
Thank you @wpears!